### ISSUE TITLE
Generating curl snippet for file in multi-part: missing @

### ISSUE THREAD

**Issue by fskexp** [usebruno/bruno#2224]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

- Create a new POST request. Set the Body as a "Multipart Form"
- Add a new File parameter
- In the left panel, open the menu on the 3 dots and click `Generate Code`
- Select `Shell-Curl`
- Observe that the command includes a parameter such as `--form file-param=/tmp/data.txt`. Instead, it should be `--form file-param=@/tmp/data.txt` so curl uses the content of the designated file.

### .bru file to reproduce the bug

```
meta {
  name: post-with-file
  type: http
  seq: 2
}

post {
  url: https://server.com/
  body: multipartForm
  auth: none
}

body:multipart-form {
  file-param: @file(/private/tmp/data.txt)
}
```

### Screenshots/Live demo link

Without the `@` (current behaviour), the request (as intercepted by a proxy) will contain the path of the file
```
--------------------------PkQw3TWuFyPky39PXxv28K
Content-Disposition: form-data; name="file-param"

/tmp/data.txt
--------------------------PkQw3TWuFyPky39PXxv28K--
```
If a `@` is manually added, then the content of the file is sent:
```
--------------------------yHQvrEAijwpPonqPCwZZ8H
Content-Disposition: form-data; name="file-param"; filename="data.txt"
Content-Type: text/plain

coucou

--------------------------yHQvrEAijwpPonqPCwZZ8H--
```

**Comment by leoferreiralima** [comment]
@fskexp i have submited a solution on a PR to this, if you can review maybe can be merged soon

**Comment by Fabiensk** [comment]
Hi, it looks good for me. I verified that in the httpsnippets library the @ is added if the parameter is ‘fileName‘

**Comment by leoferreiralima** [comment]
@Fabiensk exactly, I have added this rule on my PR, can you review please?

**Comment by sanjai0py** [comment]
Thanks for taking the time to report a bug, @fskexp. The issue you mentioned has been fixed in [PR](https://github.com/usebruno/bruno/pull/2855/), which has already been merged into the main. The fix has been available since [v1.26.0](https://github.com/usebruno/bruno/releases/tag/v1.26.0).

Closing this issue!
